### DISCUSSION TITLE
Thảo luận bài 5 6 tin học nhéeeeeeeeeeeeeeeeeeee

### DISCUSSION THREAD

**Discussion by VoHongDucC6** [VoHongDucC6/10A6-Nhom-05#11] [Announcements]
Bảng phân công nhè nhẹ:
Câu 1: Minh Tùng
Câu 2: Việt Huy
câu 3: Lê Bảo
Câu 4: Hồng Đức và Gia Hy

**Comment by giahytran** [comment]
Ngày 13/11 nộp bài 5, 6
Ngày 21/11 nộp bài 7, 8 ấy nha

**Reply by DaoMinhTung** [reply]
ok
